packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -381,7 +387,14 @@ class TapGestureRecognizer extends BaseTapGestureRecognizer {
   /// Creates a tap gesture recognizer.
   ///
   /// {@macro flutter.gestures.GestureRecognizer.supportedDevices}
-  TapGestureRecognizer({super.debugOwner, super.supportedDevices, super.allowedButtonsFilter});
+  TapGestureRecognizer({
+    super.deadline,
+    super.preAcceptSlopTolerance,
+    super.postAcceptSlopTolerance,
```

Comment:
I believe these 2 lines are the changes we need.

The rest is just exposing other `PrimaryPointerGestureRecognizer` parameters while we're at it (for consistency and configurability).

I think this is all that needs to be done (once we've updated any doc strings and tests).

Reply:
`pre/postAcceptSlopTolerance` parameters were recently added in https://github.com/flutter/flutter/commit/246d6c68341cd2b4cfbe6f67d6c856e3ed7e06c3#diff-9f1ce7ba1ae86eda2682dba82292daafdc49a66dcbc249272778140f310a1fbeR189

I'll close this PR.